src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -15,9 +19,12 @@ public User login(String login, String password) throws AuthenticationException
     }
 
     private User findByLogin(String login) {
+        logger.debug("Method findByLogin was called");
```

Comment:
Let's make our log messages informative. Include parameters in the log message to provide more context. Example: logger.debug("Method findByLogin was called with login={}", login);